includes/constants/class-wc-stripe-payment-request-button-states.php

```diff
@@ -31,6 +29,10 @@
  *    5. The following countries are not listed in WC:
  *        AC (Ascension Island), IC (Canary Islands), EA (Ceuta & Melilla), CP (Clipperton Island), DG (Diego Garcia),
  *        TA (Tristan da Cunha), XK (Kosovo).
+ *
+ * @since 5.1.0
+ *
+ * @deprecated 10.3.0 Moved to includes/constants/wc-stripe-express-checkout-button-states.php
  */
```

Comment:
To be extra-safe, we could add a `require_once` that includes the new class file. That way any dependent code that loads this file directly will continue to work as it did before.

Reply:
Good idea. Addressed it in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4871/commits/cc1222c2abb6aec6c383bf7497ca38679227bea4